src/modules/commands/discord/urbandictionary.ts

```diff
@@ -61,16 +67,114 @@ export default new Command<discord.Message>({
       )
 
     list = list.sort((a: Definition, b: Definition) => {
-      if (b.thumbsUp > a.thumbsUp) return 1
+      if (b.thumbsUp - b.thumbsDown > a.thumbsUp - a.thumbsDown) return 1
       else return -1
     })
 
-    let message = `**${wordString}:** ${list[0].word}\n\n`
-    message += `**Definition:** ${list[0].definition.replace(re, '*$1*')}\n\n`
+    let defN = 0
+    const definitionEmbed = new discord.MessageEmbed()
+      .setColor('#647CC4')
+      .setTitle(`${list[defN].word} (N°${defN + 1}/${list.length})`)
+      .setURL(list[defN].permalink)
+      .setAuthor('Urban dictionary', '', 'https://www.urbandictionary.com')
+      .setThumbnail('https://i.imgur.com/A6nvY85.png')
+      .addFields(
+        {
+          name: 'Definition',
+          value: list[defN].definition.replace(re, '*'),
+        },
+        { name: 'Example', value: list[defN].example.replace(re, '*') },
+      )
+      .addField('Written on', list[defN].writtenOn.substring(0, 11), true)
+      .addField('Thumbs up', list[defN].thumbsUp, true)
+      .addField('Thumbs down', list[defN].thumbsDown, true)
+      .setFooter('Interactive')
+      .setTimestamp()
+
+    const embedMessage = await raw.channel.send({ embed: definitionEmbed })
+
+    await embedMessage.react(constants.discord.emojis.arrow_left)
+    await embedMessage.react(constants.discord.emojis.arrow_right)
+
+    const collector = embedMessage.createReactionCollector(
+      (reaction: discord.MessageReaction, user: discord.User) =>
+        [
+          constants.discord.emojis.arrow_left,
+          constants.discord.emojis.arrow_right,
+        ].includes(reaction.emoji.name) && user.id === raw.author.id,
+      {
+        time: timeout,
+      },
+    )
+
+    setTimeout(() => {
```

Comment:
Instead of using setTimeout, listen for the event `collector` fires when it stops listening for reactions.